layouts/partials/locations/main-wrapper.njk

```diff
@@ -0,0 +1,4 @@
+<main class="grav-o-container">
+  {# Render standard block partial #}
+  {% include 'partials/blocks/main.njk' %}
+</main>
```

Comment:
Should this file be named more generically, as it really isn't specific to locations. It might come in useful on other pages where we need a `<main>` element with a `grav-o-container` class on it?

Reply:
That's fair. I'll migrate it to `blocks/wrapper-container.njk`. Does that work?